src/components/FormFieldMedia/FormFieldMedia.jsx

```diff
@@ -14,7 +14,14 @@ const FormFieldMedia = ({
   placeholder = "",
   type = "images",
   inlineButtonText = "Choose Item",
-  register = () => {},
+  register = (_args) => {},
+}: {
+  value?: string,
```

Comment:
+1 for adding types! This file is meant to be a .tsx file rather than a .jsx right?

Reply:
The types were defined here to specify the types for the typescript file which imports this component - we'd ideally like to migrate all of these files to typescript, but I think there's a whole dependency chain of files which would need to be updated here - I've opted to leave it out for now